<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sistema_Aire_Instrumentos</class>
 <widget class="QWidget" name="Sistema_Aire_Instrumentos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tiempo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Presión del Tanque Acumulador</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Presión a la Salida del Aire</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>On/Off Compresor</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Potencia Compresor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperatura a la Salida del Aire</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
